src/MarkdownTextInput.web.tsx

```diff
@@ -522,6 +528,7 @@ const MarkdownTextInput = React.forwardRef<TextInput, MarkdownTextInputProps>(
       if (autoFocus) {
         divRef.current.focus();
       }
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why do we ignore the deps here?

Reply:
I've added this so it will launch only after the first render. But after reconsideration, I think we can remove it